scheds/rust/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
 members = ["scx_lavd", "scx_bpfland", "scx_rustland", "scx_rlfifo",
-           "scx_rusty",
+           "scx_rusty", "scx_loader",
```

Comment:
Let's put this under `rust` instead of `scheds/rust`

Reply:
sure